### ISSUE TITLE
Device does not support fluxion ?

### ISSUE THREAD

**Issue by pks9862728888** [wi-fi-analyzer/fluxion#7]
My laptop supports monitor mode but fluxion givs error message that my machine does not support it.

Details of my wireless chipset: 
Driver: rtl8723be
Chipset: Realtek Semiconductor Co., Ltd. RTL873BE PCIe Wireless Network Adapter.

**Comment by GGMPL** [comment]
Does your Network Adapter is compatible with Aircrack-ng?

**Comment by pks9862728888** [comment]
yes my device is compitable with aircrack-ng....wlan0 supports monitor mode

**Comment by megagunner786** [comment]
Same. For some reason Fluxion hates Realtek cards.

**Comment by pks9862728888** [comment]
Yeah, my wifi chipset is of realtek.

**Comment by megagunner786** [comment]
I have RTL7871, supports monitor mode and injection.
Yet I just can't connect to the AP I created. I will try linset (older version of Fluxion) later on.

**Comment by azamet90** [comment]
> I have RTL7871, supports monitor mode and injection.
> Yet I just can't connect to the AP I created. I will try linset (older version of Fluxion) later on.

i cannot connect on intel wifi to AP or on realtek... all same
